### ISSUE TITLE
about dataloader

### ISSUE THREAD

**Issue by bolin12** [JDAI-CV/DCL#46]
in dataset_DCL.py 
function __getitem__()
there is :
            return  img_unswap, img_swap, label, label_swap, swap_law1, swap_law2, self.paths[item]
when i debug in train_model.py,
         inputs, labels, labels_swap, swap_law, img_names = data
why they are different items, and why the batch of inputs are twice of batch_size

can you help me find out why?

**Comment by dc3ea9f** [comment]
The usage of `collate_fn`: [url](https://pytorch.org/docs/stable/data.html#working-with-collate-fn)

During training, [`collate_fn4train`](https://github.com/JDAI-CV/DCL/blob/73ef2b794476f1143ad7cb416942f1dfbd3490b3/utils/dataset_DCL.py#L120) organize a batch data and return to the main procedure. In Line 127-128, the DL image and origin image is added to same variable.
